src/services/user/services/AdminUsers.js

```diff
@@ -278,17 +307,19 @@ class AdminUsers {
 			if (!equalIds(currentUser.schoolId, userToRemove.schoolId)) {
 				throw new Forbidden('You cannot remove users from other schools.');
 			}
-			await this.app.service('accountModel').remove(null, { query: { userId: id } });
-			return this.app.service('usersModel').remove(id);
+			// await this.app.service('accountModel').remove(null, { query: { userId: id } });
+			// return this.app.service('usersModel').remove(id);
```

Comment:
please do not remove this if you do not have a concept to not
 display deleted users